ImputationPipeline/AggregatePRSResults.wdl

```diff
@@ -220,7 +220,12 @@ task BuildHTMLReport {
 
     ## Control Sample
     \`\`\`{r control, echo = FALSE, results = "asis"}
-    kable(list(batch_control_results, expected_control_results) %>% reduce(bind_rows) %>% select(ends_with('_adjusted')) %>% add_column(sample=c('batch_control', 'expected_control'), .before=1), digits = 2)
+    control_and_expected <- list(batch_control_results, expected_control_results) %>% reduce(bind_rows) %>% select(ends_with('_adjusted'))
+    delta_frame <- control_and_expected[-1,] - control_and_expected[-nrow(control_and_expected),] %>% mutate(across(everything(), ~ round(.x, digits=2)))
+    delta_frame_colored <- delta_frame %>% mutate(across(everything(), ~ round(.x, digits=2))) %>% mutate(across(everything(), ~ kableExtra::cell_spec(.x, color=ifelse(is.na(.x) || abs(.x) > 0.12, "red", "green"))))
+    control_and_expected_rounded <- control_and_expected %>% mutate(across(everything(), ~ format(round(.x, digits=2), nsmall=2)))
+    control_table <- list(control_and_expected_rounded, delta_frame_colored) %>% reduce(bind_rows) %>% select(ends_with('_adjusted')) %>% select(order(colnames(.)))
```

Comment:
bind_rows can take the list here too.  and you shouldn't need the `select(ends_with('_adjusted'))` since you've already done that selection in line 223